examples/farm/platform/platform.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) CloudCover
+# SPDX-License-Identifier: Apache-2.0
+
+platforms:
+  - name: postgres
```

Comment:
We need to think more about what this `name` attribute is here, if we want to use it for display purpose on the UI, then it should be calls `title` & `description` for consistency.
I can't think of any other usage of this at this time... 🤔?

also, postgres, redis would be bad example. it should be `AWS EKS`, `AWS Serverless`, `GCP GKE`, `GCP Serverless`, etc.

Reply:
Ya got it. That make sense to achieve consistency